### ISSUE TITLE
Display committer information

### ISSUE THREAD

**Issue by schlick** [rowanj/gitx#346]
Feature Request: Provide the ability to display committer information (Commit + CommitDate) in the detailed view of a commit and as extra columns in the tree view.

**Comment by jeffam** [comment]
Agreed, and I'd also like to see the Date column use the original author date, rather than the commit date. I notice this when I rebase or amend commits, and the date column shows a more recent date than the Date: header in the detailed view.

Also, thanks for continuing work on this app. I really appreciate your efforts!

**Comment by mczepiel** [comment]
I can take a look at adding the column and the author date in the detail view. FWIW, I wouldn't expect this to be the primary date; I find it a lot more important to find out when something became part of a project rather than when it was written. It's easy enough to make both available though)

Anyway, I'm working on the column but running into the issue I linked where I'm somehow increasing up the webview font size.
